anabot/runtime/installation/hub/software_selection.py

```diff
@@ -216,3 +220,52 @@ def addon_check(element, app_node, local_node):
     if action_result(element)[0] == False:
         return action_result(element)
     return addon_handler_manipulate(element, app_node, local_node, True)
+
+def get_full_kernel_option_name(kernel_option):
+    if kernel_option == '4k':
+        return FULL_4k_KERNEL_OPTION
+    elif kernel_option == '64k':
+        return FULL_64k_KERNEL_OPTION
+    else:
+        return None
+
+@handle_act('/kernel_options')
+def kernel_options_handler(element, app_node, local_node):
+    chosen_kernel_option = get_attr(element, "value")
+    if chosen_kernel_option is None:
+        return NotFoundResult("No kernel options specified", where="Anabot recipe.")
+    try:
+        kernel_combo_box = getnode(local_node, "combo box",
+                                   tr(DEFAULT_KERNEL_OPTION, context="GUI|Software Selection|Kernel Options"))
+    except TimeoutError:
+        return NotFoundResult(f"Couldn't find the {DEFAULT_KERNEL_OPTION} 'Kernel Options combo box'",
```

Comment:
As mentioned earlier, NotFoundResult behaves a bit differently than possibly expected when reporting the failure. In this case, it means that you should only mention the subject that couldn't be found (`{DEFAULT_KERNEL_OPTION} 'Kernel Options combo box'`)

Reply:
Sure!